mission-control/docs/topology/references/exec.md

```diff
@@ -1,33 +1,48 @@
+---
+title: Exec
+---
+
 # <Icon name="console" /> Exec
 
 Exec Check executes a command or script file on the target host. The type of scripts executed include:
 
 - Bash scripts
 - Powershell scripts
 
-```yaml
+```yaml title="exec-lookup.yml"
+---
 apiVersion: canaries.flanksource.com/v1
-kind: Canary
+kind: Topology
 metadata:
-  name: exec-check
+  name: kubernetes-pods
 spec:
-  interval: 30
-  exec:
-  - description: "exec dummy check"
-    script: |
-      echo "hello"
-    name: exec-pass-check
-    test:
-      expr: 'results.Stdout == "hello"'
+  type: Pods
+  icon: kubernetes
```

Comment:
This example isn't very practical , using an exec plugin to create a Server component by pulling data from uname, ifconfig, etc would be more practical